### DISCUSSION TITLE
765

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#933] [Comments]
# 765

요즘 외부 개발자분과의 채용을 목적으로 한 커피챗을 하다보면 종종 받는 질문 중 하나가 CTO인 내가 지향하는 개발 문화가 무엇인지이다. 배민에 있을때 꽤 많은 팀을 옮겨다녔다. 그 중 가장 기억에 남는 팀이 있었는데, 이 팀의 만족도가 굉장히 높았다. 이 시기에는 우리팀이라면 회사의 어떤 문제든 풀 수 있을것 같다는 자신감이 있었다. 여러 기술 부채도 많이 해결했고, 여러 팀과 함께 협업하는 일도 많았다. 팀의 기술 부채 사례를 가지고 사내, 사외 공유도 많이 할 수 있었다. 당시엔 팀의 개발 문화에 대해서 크게 생각하지 않았다. 팀장님도 개발문화에 대한 언급을 그렇게 많이 하지 않으셨고, 가능하면 팀원들과 함께 회사의 여러 문제를 해결하는 것에 집중하셨다. 그러니깐, 팀의 개발문화를 위해 어떠한 액션..

https://jojoldu.tistory.com/765

**Comment by AppleCEO** [comment]
넘넘 좋아요!

**Comment by bysxx** [comment]
정말 좋은 내용인 것 같습니다..! 좋은 개발자들이 모이면 자연스레 좋은 개발 문화가 생긴다는 점이 정말 공감됩니다!

**Comment by Invidam** [comment]
노자가 말한 "도라고 말할 수 있으면 도가 아니다"라는 내용이 향로님이 말한 문화와 비슷하게 생각이 드네요.
